lib/mote_sms/delivery_job.rb

```diff
@@ -0,0 +1,13 @@
+module MoteSMS
+  class DeliveryJob < ActiveJob::Base
+    queue_as :default
```

Comment:
can we make the queue configurable like in ActionMailer / also to be provided as an option into `deliver_later`?